jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientTLSTest.java

```diff
@@ -1377,8 +1377,6 @@ public void testForcedNonDomainSNI() throws Exception
     @EnabledForJreRange(max = JRE.JAVA_16, disabledReason = "Since Java 17, SNI host names can only have letter|digit|hyphen characters.")
     public void testForcedNonDomainSNIWithIPv6() throws Exception
     {
-        Assumptions.assumeTrue(Net.isIpv6InterfaceAvailable());
-
```

Comment:
This (and all other occurrences) should remain.
If IPv6 is available in the new Jenkins setup, then this assumption will pass and the test will run.

Reply:
I agree with @sbordet - this assumption is also valid for users that build on machines that don't have IPv6 present/available.

Reply:
It's temporary and will be reverted as I wanted to be sure ipv6 tests were running! 
Maybe if someone could argue "blabla ipv6 tests are disabled" 🤣